CONFIGURATION.md

```diff
@@ -66,7 +66,7 @@ ssl_key                                  |  *  |                 |
 ssl.certificate.location                 |  *  |                 |               | low        | Path to client's public key (PEM) used for authentication. <br>*Type: string*
 ssl.certificate.pem                      |  *  |                 |               | low        | Client's public key string (PEM format) used for authentication. <br>*Type: string*
 ssl_certificate                          |  *  |                 |               | low        | Client's public key as set by rd_kafka_conf_set_ssl_cert() <br>*Type: see dedicated API*
-ssl.ca.location                          |  *  |                 |               | low        | File or directory path to CA certificate(s) for verifying the broker's key. <br>*Type: string*
+ssl.ca.location                          |  *  |                 |               | low        | File or directory path to CA certificate(s) for verifying the broker's key. Defaults: On Windows the system's CA certificates are automatically looked up in the Windows Root certificate store. On Mac OSX it is recommended to install openssl using Homebrew, to provide CA certificates. On Linux install the distribution's ca-certificates package. If OpenSSL is statically linked or `ssl.ca.location` is set to `probe` a list of standard paths will be probed and the first one found will be used as the default CA certificate location path. If OpenSSL is dynamically linked the OpenSSL library's default path will be used (see `OPENSSLDIR` in `openssl version -a`). <br>*Type: string*
```

Comment:
Are the homebrew instructions always necessary or only when openssl is statically linked.

Reply:
Always, OSX does not provide any ca cert fels, it has its keystore, but when installing openssl thru brew I believe it either downloads proper ca-certs, or extracts certs from the keystore and puts them on disk.

Reply:
Surely OSX ships with some CA certs otherwise using Safari would be quite unpleasent

Reply:
They are in the keystore, not as files.